app/controllers/api/base_controller/normalizer.rb

```diff
@@ -24,7 +24,8 @@ def normalize_hash(type, obj, opts = {})
         is_ar = obj.kind_of?(ActiveRecord::Base)
         attrs.each do |k|
           next if Api.encrypted_attribute?(k) && api_resource_action_options.exclude?("include_encrypted_attributes")
-          next if is_ar ? !obj.respond_to?(k) : !obj.key?(k)
+          next if is_ar ? !obj.respond_to?(k) : !obj.try(:key?, k)
```

Comment:
So, service service isn't AR::Base and also not a hashlike object.  Is this a new problem or has it always been a problem?

Is there a test for this method we can add to?